apps/nrm/views/partials/summary-table-potential-victim.html

```diff
@@ -88,7 +88,7 @@ <h2 class="details-header">{{#t}}pages.confirm.sections.potential-victim.header{
       {{#values.pv-gender}}
       <tr>
         <td>{{#t}}pages.confirm.fields.pv-gender.label{{/t}}</td>
-        <td class="no-white-space">
+        <td class="no-white-space capitalise-first-letter remove-dashes">
```

Comment:
The remove-dashes class here will not allow dashes to be shown for the gender value.
Once this is merged then the next PR should not have any issue with dashes being shown.
@sulthan-ahmed